README.md

```diff
@@ -178,9 +169,7 @@ kubectl create -f wavefront-with-http-proxy.yaml
 
 # Upgrade
 
-The Operator installation process is idempotent,
-so these commands should look familiar
-to what you did during installation.
+Upgrade from one operator version to another? Upgrade from previous installation methods? From what state, perform what action to result in what expected outcome? Idempotent is a reason why an approach works, but is mostly ancillary to the user's goal. We need more detail here.
```

Comment:
This is just a comment that needs to be addressed and not an actual modification. So adding my comment for tracking purposes.

Reply:
Thanks, I had a hard time getting github to let me comment without a change to comment on. :)

Reply:
I took a stab at updating it